### PR TITLE
console: additional updates for dynamic plugin GA

### PR DESCRIPTION
**Pull request by spadgett**
Remove statements saying that plugins are not a public API now that it's GA and clean up some formatting.

/assign @jhadvig

### CODE REVIEW

**Comment by dhellmann** [comment]
@spadgett this change looks like a relatively minor update that aged out so the bot closed it. Should it be reopened so we can merge it?

**Comment by spadgett** [comment]
/reopen

Hey @jhadvig do you mind reviewing?

File: enhancements/console/dynamic-plugins.md [hunk 1]
```diff
@@ -71,9 +71,6 @@ contributing to a single monorepo.
 
 ### Non-Goals
 
-* Initially we don't plan to make this a public API. The target use is for Red
-  Hat operators. We might reevaluate later when dynamic plugins are more
-  mature.
```
Comment (spadgett) [inline enhancements/console/dynamic-plugins.md:76]: This is the important change... Dynamic plugins are a public API since 4.12.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. It does not appear to be linked to a valid Jira ticket. Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. It does not appear to be linked to a valid Jira ticket. Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**